static/app/components/devtoolbar/components/releases/useToolbarRelease.tsx

```diff
@@ -0,0 +1,24 @@
+import type {Release} from 'sentry/types/release';
+import {useApiQuery} from 'sentry/utils/queryClient';
```

Comment:
If this switches to `from '../../hooks/useFetchApiData'` then this can also change in releasesPanel.tsx:
```
- <ReleaseHeader release={data[0]} orgSlug={organizationSlug} />
+ <ReleaseHeader release={data} orgSlug={organizationSlug} />
```

eventually we'll have to remove any `from 'sentry/...` imports. This one needs to be replaced because `queryClient.tsx` imports from stores and organization and lots of stuff like that.